.gitignore

```diff
@@ -1,17 +1,17 @@
 node_modules
 npm-debug*
 
-actions.js
-Client.js
-components.js
-createTest.js
-merge.js
-navigationMiddleware.js
-pageUtils.js
-plugins.js
-reducer.js
-router.js
-Server.js
-shouldGoBack.js
-url.js
-platform.js
+/actions.js
```

Comment:
It's not you -- git does the right thing but not atom... but this makes both happy.